apps/bfd-model/bfd-model-rif-samples/src/main/java/gov/cms/bfd/model/rif/samples/StaticRifFile.java

```diff
@@ -22,25 +26,29 @@ public StaticRifFile(StaticRifResource staticRifResource) {
     this.staticRifResource = staticRifResource;
   }
 
-  /** @see gov.cms.bfd.model.rif.RifFile#getFileType() */
+  /** Gets the file type of the static rif resource as a {@link RifFileType}. */
```

Comment:
done

Reply:
You didn't update the Override methods to use {@inheritdoc}.

Reply:
let me know if that was wrong

Reply:
Yeah still not quite right; overrides _only_ need inheritdoc because they are just taking the documentation of their inherited method/class/etc. It should look like this (example):
![image](https://user-images.githubusercontent.com/1200656/215132948-8b4b7ac9-19cb-4c08-ad75-cf1a87f0c3cb.png)

Reply:
done